packages/react-components/react-menu/src/components/MenuItem/useMenuItemStyles.ts

```diff
@@ -18,8 +18,22 @@ export const menuItemClassNames: SlotClassNames<MenuItemSlots> = {
 
 const useStyles = makeStyles({
   focusIndicator: createFocusOutlineStyle(),
+  // TODO: this should be extracted to another package
+  resetButton: {
```

Comment:
the default element for the menu item is a `div` these styles are required if users start using `button` correct ?

Reply:
Completely correct

Reply:
Done ✅, now I'm checking if it's a native `button` before applying the `resetButton` style